clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -32,36 +53,13 @@ spec:
       # Default: Start with 2Gi and add 2Gi for each 1K nodes.
       memory: {{MultiplyInt $PROMETHEUS_MEMORY_LIMIT_FACTOR (AddInt 1 (DivideInt .Nodes 1000))}}Gi
       {{end}}
-  ruleSelector:
-    matchLabels:
-      prometheus: k8s
+  ruleNamespaceSelector: {}
+  ruleSelector: {}
   securityContext:
     fsGroup: 2000
     runAsNonRoot: true
     runAsUser: 1000
-  tolerations:
-  - key: "monitoring"
-    operator: "Exists"
-    effect: "NoSchedule"
   serviceAccountName: prometheus-k8s
   serviceMonitorNamespaceSelector: {}
   serviceMonitorSelector: {}
-  podMonitorNamespaceSelector: {}
-  podMonitorSelector: {}
-  version: v2.25.0
-  retention: 7d
-  storage:
-    volumeClaimTemplate:
-      spec:
-        storageClassName: ssd
-        resources:
-          requests:
-            # Start with 10Gi, add 10Gi for each 1K nodes.
-            storage: {{MultiplyInt 10 (AddInt 1 (DivideInt .Nodes 1000))}}Gi
```

Comment:
Please do not remove the storage sizing logic.

Reply:
https://github.com/kubernetes/perf-tests/pull/2040/commits/ec88962d79cc233adfb94807b34d5d6bfe591d91
This commit adds this back , and it's part of this PR.  I will update this file after I open a separate PR for Azure-Disk support